flink-table/flink-table-common/src/main/java/org/apache/flink/table/api/dataview/MapView.java

```diff
@@ -256,25 +252,8 @@ public static DataType newMapViewDataType(DataType keyDataType, DataType valueDa
                         "map", DataTypes.MAP(keyDataType, valueDataType).bridgedTo(Map.class)));
     }
 
-    // --------------------------------------------------------------------------------------------
-    // Legacy
-    // --------------------------------------------------------------------------------------------
-
-    @Deprecated public transient TypeInformation<?> keyType;
-
-    @Deprecated public transient TypeInformation<?> valueType;
-
     /**
-     * Creates a {@link MapView} with the specified key and value types.
-     *
-     * @param keyType The type of keys of the map view.
-     * @param valueType The type of values of the map view.
-     * @deprecated This method uses the old type system. Please use a {@link DataTypeHint} instead
-     *     if the reflective type extraction is not successful.
+     * {@code MapView(TypeInformation<?> keyType, TypeInformation<?> valueType)} method was
+     * deprecated and removed. Please use a {@link DataTypeHint} instead.
```

Comment:
This now floats as a comment in the file without being attached to the class or a method which only makes it useful when viewing sources. I wonder if it should be part of the javadoc for the class.

Reply:
I moved it to the javadoc for the class.